fiftyone/utils/cvat.py

```diff
@@ -3939,6 +3947,48 @@ def _upload_annotations(
             num_uploaded_tags = len(anno_resp["tags"])
             num_uploaded_tracks = len(anno_resp["tracks"])
 
+        server_id_map = self._map_cvat_server_id(anno_resp, attr_id_map)
+
+        return server_id_map
+
+    def _map_cvat_server_id(self, anno_resp, attr_id_map):
```

Comment:
How about `_create_server_id_map()`? More consistent with the other method names